DevGuideExamples/FACE/two-topic/CMakeLists.txt

```
@@ -0,0 +1,45 @@
+cmake_minimum_required(VERSION 3.3...3.27)
+project(opendds_stock CXX)
```

This name should mention it's using FACE since there's another "stock quoter" that's not using FACE.

```suggestion
project(opendds_face_stock CXX)
```